webclient/src/dialogs/RegistrationDialog/RegistrationDialog.tsx

```diff
@@ -26,8 +26,8 @@ const RegistrationDialog = ({ classes, handleClose, isOpen }: any) => {
           </IconButton>
         ) : null}
       </DialogTitle>
-      <DialogContent>
-        <RegisterForm onSubmit={handleOnClose}></RegisterForm>
+      <DialogContent className="dialog-content">
```

Comment:
We've been inconsistent on naming with capitals, dashes, and such. Should probably unify at some point